testnets/chain4energytestnet/chain.json

```diff
@@ -0,0 +1,89 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "chain4energy testnet",
```

Comment:
please change to 'chain4energytestnet' (one word, to match the folder name exactly)